### DISCUSSION TITLE
<q-input  no padding, no  margin, no indentation, no emphasizing, no background?

### DISCUSSION THREAD

**Discussion by dixiPro** [quasarframework/quasar#15310] [General - Components / Directives / etc]
<q-btn
How can I do that there were no padding, no  margin, no indentation, no emphasizing. A small field for entering? I have already reviewed  the documentation three times.

I tried :dense="true"
I tried class="q-ma-none q-pa-none"
I tried input-class="q-ma-none q-pa-none"

Is there anyone who can answer such a simple question?

**Comment by MilosPaunovic** [comment]
https://codepen.io/milospaunovic/pen/rNrdbRe?editors=101

**Reply by flaviofl** [reply]
Adding **dense** to the q-input resolved for me

```javascript
<q-input
    outlined
    autogrow
    dense
/>
```
Thank you!
